reference/release-notes/1.24.0.md

```diff
@@ -0,0 +1,101 @@
+---
+orphan: true
+---
+# 1.24.0
+
+These release notes cover new features and changes in Anbox Cloud 1.24.0.
+
+Anbox Cloud 1.24.0 is a minor release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See the [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+### Updates to components
+
+* Anbox Cloud includes and uses the following latest versions for components it uses:
+  - LXC version 6.0.2<!--AC-2812-->
+  - Mesa driver version 24.2.4<!--AC-2800-->
+  - VirGL renderer version 1.9.0.<!--AC-2800-->
+
+### Security
+
+* We have enhanced security with a minimum version of TLS 1.3 as a requirement for all services working with Anbox Cloud. TLS 1.3 is also enabled by default for the Anbox Stream Agent and Gateway.<!--AC-2728 and AC-2654-->
+
+### Anbox Management Service (AMS)
+
+* We have increased the maximum number of queued operations in AMS to allow better performance.<!--AC-2831-->
+* AMS supports querying the image variant (`android` or `aaos`) for Anbox Cloud images.<!--AC-2757-->
+* The web client, for example, the Anbox Cloud dashboard can communicate with multiple AMS instances.<!--AC-2674-->
+
+### Android
+
+* This release includes Android 14 based AOSP and AAOS images.
+* We now support using the lock screen in Anbox Cloud([LP 1983426](https://bugs.launchpad.net/anbox-cloud/+bug/1983426)).
+* The out of band v2 feature in Anbox Cloud requires the Android app to be running to receive broadcasts. If the app is in the [cached state](https://developer.android.com/guide/components/activities/process-lifecycle), the system places [context-registered broadcasts in a queue]((https://developer.android.com/develop/background-work/background-tasks/broadcasts#android-14)) and hence the app may not receive broadcasts immediately as it would when it is running.<!--AC-2852-->
```

Comment:
The out of band v2 feature in Anbox Cloud requires the Android app to be running to receive broadcasts when using the Android 14 LXD image. In short, this is Android 14 specific.
That implies if people are using Android 12 or Android 13 LXD images, apps can stay cache state and the out of band v2 feature worked fine.